ExtLibs/MV04.Camera/CameraHandler.cs

```diff
@@ -723,8 +737,10 @@ public bool StartTracking(Point? trackPos)
                 _trackPos.X = Constrain(_trackPos.X, 0, 1280);
                 _trackPos.Y = Constrain(_trackPos.Y, 0, 720);
 
+                MavCmdSetTrackingMode(CameraControl.mav_comm, CameraControl.ackCb, _trackPos.X, _trackPos.Y, (int)TrackerMode.TrackOnPosition, 0);
```

Comment:
Nálam a TrackerMode.Track is működött, de miért nem maradt meg a mav_error érték a visszatérési érték feltételeként?